### ISSUE TITLE
Ankicommunity Discord Server 

### ISSUE THREAD

**Issue by luwol03** [ankicommunity/ankicommunity.github.io#5]
Hello guys,

I propose to create a discord server for the Ankicommunity, as I think many people use discord. I would also say that the discord mesages and pings are much easier than gitter messages without mobile Push notifications. I just opened a issue here, as Vikash Kothary suggested. I would be happy if you could vote with reactions to this issue.

**Comment by kuklinistvan** [comment]
Hello,

I'm really sorry to say that but I'm not sure I'm 100% convinced. I'm affraid it would split up the communication of the community to two different spaces. Either I would close (cap) Gitter at least or just stay there. 

Also, in Gitter, it is very easy to join the conversation only with a Github account, whereas Discord requires some more clicks of setup. It is not that hard, but I'm sure it will decrease the number of participants. 

At the very least we should create a poll on this with everyone on Gitter, I think.

I do use Discord, though, but - ~~what do you propose, what are the benefits of that platform that would make the move worth it anyway?~~
(sorry, you did write that, did not read carefully)

However we decide, thank you for the proposal!

**Comment by kuklinistvan** [comment]
On Gitter I get both emails and  notifications from the browser.

**Comment by VikashKothary** [comment]
Hi all,

**TL;DR; My stance is currently: creating an AnkiCommunity Discord Server is a good idea, but just not right now.**

I want to start by reiterating what @kuklinistvan said: @luwol03, thanks for your proposal and your interest in AnkiCommunity.

So I want to add to the conversation which is why I suggested we create this ticket. So I use Discord a lot and I really like it. There's a bunch of benefits to Discord over Glitter and I'll list a few here:
- Discord supports Voice/Video Chat
- Discord supports multiple channels for different projects 
- Discord has a much for mature bot ecosystem to add additional functionality
- Discord has a better client for Android (as I assume iOS but I can't attest to that fact)
- Discord supports better user management including user permissions, groups, etc.
- And probably more

That being said, I don't think any of those features are needed at this point. Glitter is simple and works well enough. We currently have 27 users in the Glitter chat therefore I don't think it's worth the time/energy cost to migrate the users. @kuklinistvan also makes a good point that a migration from Glitter to Discord will probably have us losing users.

I also have a personal philosophical reason for staying on Glitter that make me think that right now is not the right time. My current goal is to bring all the projects together and align the different initiatives. I believe Glitter is better for that right now because a single chat means everyone is visible of all the conversation going on right now. Adding complexity will probably silo the conversations which isn't good right now in the development of the community. 

I see a need for Discord in the future (or somnething like it) when we need to scale but I don't think right now is that time.

That's my 2 cents. Let me know what you think.

**Comment by luwol03** [comment]
Closing in favor of a gitter chat.
